templates/components/loaders/typescript/file/loader.ts

```diff
@@ -1,7 +1,7 @@
 import {
   FILE_EXT_TO_READER,
   SimpleDirectoryReader,
-} from "llamaindex/readers/SimpleDirectoryReader";
+} from "llamaindex/readers/index";
```

Comment:
weird as https://github.com/run-llama/LlamaIndexTS/blob/c01502fb8460aee4350481035a5d224b3aa807a8/examples/readers/src/custom-simple-directory-reader.ts#L3-L7 is using this import and the examples have a type check running
```suggestion
} from "llamaindex";
```

Reply:
just tested it adding the type-checks to the readers: https://github.com/run-llama/LlamaIndexTS/pull/1420 - works